build.go

```diff
@@ -212,6 +212,40 @@ func (b *Build) GetConsoleOutput() string {
 	return content
 }
 
+// GetProgressiveText gets progressive text with the param in previous response header.
+func (b *Build) GetProgressiveText(start int64) (string, string, error) {
+	startStr := strconv.FormatInt(start, 10)
+	url := b.Base + "/logText/progressiveText"
+
+	var content string
+	querymap := make(map[string]string)
+	querymap["start"] = startStr
+
+	response, err := b.Jenkins.Requester.Get(url, &content, querymap)
+	if err != nil {
+		return "", "", err
+	}
+	textSize := response.Header.Get("X-Text-Size")
```

Comment:
What is `textSize` for? Which case need it?

Reply:
The textSize is the length of the log we get, we can use it as next start so that we can get progressive log without repeat.